Sources/Swift/Integrations/SessionReplay/SentrySessionReplay.swift

```diff
@@ -61,17 +64,7 @@ class SentrySessionReplay: NSObject {
     }
 
     func start(rootView: UIView, fullSession: Bool) {
-        guard !isRunning else { return }
```

Comment:
m: I believe we should keep this, it will avoid problems described in the next comment.

Reply:
I can see how we'd want to keep this to essentially debounce multiple calls to `start`.

Reply:
I added this check back in, so I think both of your comments are addressed by that? LMK if there's anything else needed.